ai21/models/chat/document_schema.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import annotations
+from typing import Optional
+from pydantic import Field
+from ai21.models.ai21_base_model import AI21BaseModel
+
+
+class DocumentSchema(AI21BaseModel):
+    id: Optional[str] = Field(None, description="Id of the document", max_length=128)
```

Comment:
This length validation can change in the backend and thus you'll have to upgrade the client to support it. I would avoid adding client side validations